clients/unshaded/src/test/java/tachyon/client/netty/ClientHandlerTest.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.client.netty;
+
+import java.io.IOException;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.mockito.Mockito;
+
+import io.netty.channel.ChannelHandlerContext;
+
+import tachyon.network.protocol.RPCBlockReadRequest;
+import tachyon.network.protocol.RPCBlockReadResponse;
+import tachyon.network.protocol.RPCMessage;
+import tachyon.network.protocol.RPCResponse;
+import tachyon.network.protocol.databuffer.DataBuffer;
+
+public class ClientHandlerTest {
+
+  @Rule
+  public final ExpectedException mThrown = ExpectedException.none();
+
```

Comment:
Since every test creates a `ClientHandler` and almost all tests mock a `ChannelHandlerContext`, consider setting them up in a `@Before` method.